Solutions/DNS Domain Solution for Microsoft Sentinel/Workbooks/DNSSolutionWorkbook.json

```diff
@@ -0,0 +1,951 @@
+              "query": "let maxSummarizedTime = toscalar (\r\n    union isfuzzy=true \r\n        (\r\n            DNS_Summarized_Logs_sourceInfo_CL\r\n                | where EventTime_t >= {TimeRange:start}\r\n                | summarize max_TimeGenerated=max(EventTime_t)\r\n                | extend max_TimeGenerated = datetime_add('minute',10,max_TimeGenerated)\r\n        ),\r\n        (\r\n            print({TimeRange:start})\r\n            | extend max_TimeGenerated = print_0\r\n            | project max_TimeGenerated\r\n        )\r\n        | summarize maxTimeGenerated = max(max_TimeGenerated) \r\n    );\r\nunion isfuzzy=true \r\n    (\r\n        _Im_Dns(starttime=todatetime(maxSummarizedTime), endtime=now())\r\n\t  | extend EventProduct = strcat(EventVendor,\"-\",EventProduct)\r\n        | summarize Instances=toint(count()) by EventProduct, bin(TimeGenerated, 6h)\r\n    ),\r\n    (\r\n        DNS_Summarized_Logs_sourceInfo_CL\r\n        | where EventTime_t >= {TimeRange:start}\r\n        | where isnotempty(EventProduct_s)\r\n        | summarize Instances=toint(sum(count__d)) by EventProduct=EventProduct_s, TimeGenerated=bin(EventTime_t, 6h)\r\n    )\r\n    | summarize Instances = sum(Instances) by EventProduct, bin(TimeGenerated, 6h)",
+              "size": 1,
+              "title": "Events by Source Type over time",
+              "timeContextFromParameter": "TimeRange",
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "visualization": "timechart"
+            },
+            "customWidth": "50",
+            "name": "query - 0",
+            "styleSettings": {
+              "margin": "0",
+              "padding": "0",
+              "showBorder": true
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "let maxSummarizedTime = toscalar (\r\n    union isfuzzy=true \r\n        (\r\n            DNS_Summarized_Logs_sourceInfo4_CL\r\n                | where EventTime_t >= {TimeRange:start}\r\n                | summarize max_TimeGenerated=max(EventTime_t)\r\n                | extend max_TimeGenerated = datetime_add('minute',10,max_TimeGenerated)\r\n        ),\r\n        (\r\n            print({TimeRange:start})\r\n            | extend max_TimeGenerated = print_0\r\n            | project max_TimeGenerated\r\n        )\r\n        | summarize maxTimeGenerated = max(max_TimeGenerated) \r\n    );\r\nunion isfuzzy=true \r\n    (\r\n        _Im_Dns(starttime=todatetime(maxSummarizedTime), endtime=now())\r\n        | summarize Instances=toint(count()) by bin(TimeGenerated, 6h)\r\n    ),\r\n    (\r\n        DNS_Summarized_Logs_sourceInfo_CL\r\n        | where TimeGenerated >= {TimeRange:start}\r\n        | summarize Instances=toint(sum(count__d)) by TimeGenerated=bin(EventTime_t,6h)\r\n    )\r\n    | summarize Instances = sum(Instances) by bin(TimeGenerated, 6h)",
```

Comment:
Is the table name correct? Shouldn't it be DNS_Summarized_Logs_sourceInfo_CL?

Reply:
corrected